ui/src/app/completed/controllers/TableCtrl.js

```diff
@@ -8,7 +8,24 @@
     $scope.dlink = {link: ''};
     $scope.downloads = [];
 
+    var formatBytes = function(bytes, decimals) {
```

Comment:
I noticed this method has been declared in multiple places. Can't we create a common function?